packages/riipen-ui-docs/src/pages/components/list/Simple.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import List from "@riipen-ui/components/List";
+import ListItem from "@riipen-ui/components/ListItem";
+
+export default function Example() {
+  return (
+    <List>
+      <ListItem>Item 1</ListItem>
+      <ListItem disabled>Item 2</ListItem>
```

Comment:
This `disabled` prop seems weird to me. Why is the list item disabled? Shouldn't it be the _child_ of the list item if any that is disabled in some way. Will check out the actual component and provide more comments.